rllib/utils/test_utils.py

```diff
@@ -1481,7 +1481,7 @@ def check_supported_spaces(
     ]
 
     # TODO(Artur): Add back tf2 once we CNNs there
-    rlmodule_supported_frameworks = {"torch"}
+    rlmodule_supported_frameworks = "torch"
```

Comment:
doesn't this need to be a list?

Reply:
Also, we do now support CNNs in tf2, so we should resolve this TODO.